inc/namespace.php

```diff
@@ -100,7 +100,7 @@ function fetch_data_for_attachment( int $id ) {
 			$responses['labels'] = $labels_response['Labels'];
 		} catch ( Exception $e ) {
 			trigger_error( $e->getMessage(), E_USER_WARNING );
-			$responses['labels'] = new WP_Error( 'aws-error', $e->getMessage() );
+			$responses['labels'] = new WP_Error( 'aws-rekognition-error', $e->getMessage() );
```

Comment:
Since we're in here, can we ignore these phpcs warnings on the trigger_error? They'll go away with HMCS 1.0, but we're not using that anyway.